mslib/msui/constants.py

```diff
@@ -37,8 +37,9 @@
 if '://' in MSUI_CONFIG_PATH:
     try:
         _fs = fs.open_fs(MSUI_CONFIG_PATH)
-    except fs.errors.CreateFailed:
         _fs.makedirs(MSUI_CONFIG_PATH)
```

Comment:
This would create a directory tree like MSUI_CONFIG_PATH inside of MSUI_CONFIG_PATH, i.e.:
```
$ tree test-fs
test-fs

0 directories, 0 files
$ python3
Python 3.11.7 | packaged by conda-forge | (main, Dec 23 2023, 14:43:09) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import fs
>>> a = fs.open_fs("osfs://~/Playground/test-fs")
>>> b = a.makedirs("osfs://~/Playground/test-fs")
>>> 
$ tree test-fs
test-fs
`-- osfs:
    `-- ~
        `-- Playground
            `-- test-fs

5 directories, 0 files
```

I don't think that is what is intended to happen here. Instead, I think the wanted result is that of `fs.open_fs(MSUI_CONFIG_PATH, create=True)`.

And I think the entire `try:` block can be replaced with just that (i.e. if there is an exception it should crash MSS and not just be logged as an error, since I'd assume there is no reasonable path forward if the config directory cannot be used).

Reply:
I think this can be radically simplified, I'll open a PR to demonstrate.